WooCommerce/Classes/ViewRelated/ReusableViews/ButtonActivityIndicator.swift

```diff
@@ -25,11 +25,13 @@ final class ButtonActivityIndicator: UIButton {
         }
         indicator.isUserInteractionEnabled = false
         indicator.center = CGPoint(x: self.bounds.size.width/2, y: self.bounds.size.height/2)
+        indicator.hidesWhenStopped = true
+        // If `hideActivityIndicator()` is called immediately after this method, it will find the indicator and remove it
+        addSubview(indicator)
```

Comment:
There was a bug where, if we called `hideActivityIndicator()` shortly after the `showActivityIndicator()` it would have no effect, since the indicator was added as subview after 0.3 seconds, and the hide method searched for this subview